platform/barefoot/sonic-platform-modules-ingrasys/debian/sonic-platform-ingrasys-s9280-64x.prerm

```diff
@@ -0,0 +1,21 @@
+# Automatically added by dh_systemd_start
+if [ -d /run/systemd/system ]; then
+    deb-systemd-invoke stop s9280-32x-monitor.service >/dev/null
+    deb-systemd-invoke stop bfn.service >/dev/null
+    deb-systemd-invoke stop qsfp-monitor.service >/dev/null
+fi
+# End automatically added section
+# Automatically added by dh_installinit
+if [ -x "/etc/init.d/s9280-32x-monitor" ]; then
+    invoke-rc.d s9280-32x-monitor stop || exit $?
+fi
+if [ -x "/etc/init.d/bfn" ]; then
+    invoke-rc.d bfn stop || exit $?
+fi
+if [ -x "/etc/init.d/qsfp-monitor" ]; then
+    invoke-rc.d qsfp-monitor stop || exit $?
+fi
```

Comment:
please do not add, this is automatically generated file.

Reply:
We will remove all the automatically generated files, thanks.